liota/entities/registered_entity.py

```diff
@@ -36,6 +36,16 @@ def __init__(self, ref_entity, ref_dcc, reg_entity_id):
         self.ref_entity = ref_entity
         self.ref_dcc = ref_dcc
         self.reg_entity_id = reg_entity_id
+        self.parent = None
```

Comment:
I don't see the requirement of defining parent attribute and the set_parent method. We can just directly create the relationship as per yesterday's discussion we don't require the parent to be part of the registered_entity. We can directly make the relationship call with help of reg_entity_id and reg_child_id. In the case of metric as of know, you'll have to set the reg_entity_id of the parent to reg_metric_entity_id so that we can invoke start_collecting method on it.